packages/staking/src/features/drawer/StakePoolConfirmation.tsx

```diff
@@ -365,17 +379,27 @@ export const StakePoolConfirmationFooter = ({ popupView }: StakePoolConfirmation
   }, [isConfirmingTx, isInMemory, keyAgentType, popupView, t]);
 
   return (
-    <div className={styles.actions}>
-      <Button
-        data-testid="stake-pool-confirmation-btn"
-        disabled={isBuildingTx || !!stakingError}
-        loading={isConfirmingTx || isBuildingTx}
-        onClick={handleConfirmation}
-        className={styles.confirmBtn}
-        size="large"
-      >
-        {confirmLabel}
-      </Button>
-    </div>
+    <>
+      <div className={styles.actions}>
+        <Button
+          data-testid="stake-pool-confirmation-btn"
+          disabled={isBuildingTx || !!stakingError}
+          loading={isConfirmingTx || isBuildingTx}
+          onClick={handleConfirmation}
+          className={styles.confirmBtn}
+          size="large"
+        >
+          {confirmLabel}
+        </Button>
+      </div>
+      <PoolsManagementModal
+        type={
+          confirmationModalVisible.adjustment ? PoolsManagementModalType.ADJUSTMENT : PoolsManagementModalType.REDUCTION
+        }
+        visible={confirmationModalVisible.reduction || confirmationModalVisible.adjustment}
```

Comment:
Also based on the previous comment we can do:

```suggestion
        type={openPoolManagementModal}
        visible={!!openPoolManagementModal}
```

Reply:
Fixed on c61b8ac